libcaf_net/src/multiplexer.cpp

```diff
@@ -106,7 +117,19 @@ ptrdiff_t multiplexer::index_of(const socket_manager_ptr& mgr) {
   return i == last ? -1 : std::distance(first, i);
 }
 
+middleman& multiplexer::owner() {
```

Comment:
Wouldn't `middleman` or at least `mm` be more clear?

Reply:
I prefer `owner` here, because it also makes the lifetime clear. Once the owner (middleman) is gone, the multiplexer goes as well. This is a stronger relation than just "the multiplexer knows the middleman".